### DISCUSSION TITLE
use public solar production data

### DISCUSSION THREAD

**Discussion by quietrider** [electricitymaps/electricitymaps-contrib#4104] [Ideas] [answered]
https://monitoringpublic.solaredge.com/solaredge-web/p/home/public?locale=en_US 
Can this data be used to shape behavior? 
Track how much carbon is avoided by manual efforts like timing the dryer for when solar is producing vs grid. Build a community around a game with prize$ for Top 100 Carbon Intensity Minimizers. Installers, politicians, subsidies flow from the data. Solar and timing decarbonize a home/biz faster than decarbonizing the grid.

**Comment by madsnedergaard** [comment, marked as answer]
Thanks for opening this discussion :)
While that data and idea seems interesting, it's also very much outside the scope of what electricityMap is and does. I don't see this as something we could do, but it's a nice idea.
